index.html

```diff
@@ -5,26 +5,53 @@
     <title>My website</title>
   </head>
   <body>
-    <form action="index.php" method="post">
+    <form action="index.php" method="post" enctype="multipart/form-data">
       <label for="username"> username</label>
       <input
         type="text"
         id="username"
         placeholder="Hassan"
-        minlength="6"
-        maxlength="15"
+        min="6"
+        max="15"
         required
       />
-
       <label for="password">password</label>
+      <input type="password" id="password" min="6" max="15" required />
+      ><br />
+      <label for="email">email</label>
+      <input type="email" id="email" placeholder="hassanfd@gmail.com" /> <br />
```

Comment:
The place holder shouldn't be your email. Use just "email" or "example@gmail.com"